stable/unbound/templates/service.yaml

```diff
@@ -0,0 +1,18 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ template "unbound.fullname" . }}
+  labels:
+    app: {{ template "unbound.name" . }}
+    chart: {{ template "unbound.chart" . }}
+    release: {{ .Release.Name  }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    app: {{ template "unbound.name" . }}
+    release: {{ .Release.Name }}
+  ports:
```

Comment:
Make list indentation level consistent. Since you did not indent lists in other places, don't do it here either.